src/App.ts

```diff
@@ -243,10 +269,28 @@ export default class App {
       ...installerOptions,
     };
 
+    if (
+      this.developerMode &&
+      this.installerOptions &&
+      (typeof this.installerOptions.callbackOptions === 'undefined' ||
+        (typeof this.installerOptions.callbackOptions !== 'undefined' &&
+          typeof this.installerOptions.callbackOptions.failure === 'undefined'))
+    ) {
+      // add a custom failure callback for Developer Mode in case they are using OAuth
+      this.logger.debug('adding Developer Mode custom OAuth failure handler');
+      this.installerOptions.callbackOptions = {
+        failure: (error, _installOptions, _req, res) => {
+          this.logger.debug(error);
+          res.writeHead(500, { 'Content-Type': 'text/html' });
```

Comment:
This worked in a express based bolt app. Need to test it with `socketMode` OAuth app (uses `createServer` under the hood from built in `http` node module)

Reply:
It was a bug! Fixed now!